pkg/cluster/manifests/charts/capi-cluster-autoscaler/templates/rbac.yaml

```diff
@@ -1,3 +1,11 @@
+apiVersion: v1
```

Comment:
As mentioned elsewhere, if the controller no longer reads service accounts, you can remove `serviceaccounts-reader` and related resources.

Reply:
Please check the comment above.